### PR TITLE
Issue #4752: Add missing .htaccess rules for PHP 7

### LINKED ISSUE #4752: Add missing .htaccess rules for mod_php7
Some of the `.htaccess` rules for PHP 5 are not applied for PHP 7:

```
# Override PHP settings that cannot be changed at runtime. See
# settings.php and backdrop_environment_initialize() in includes/bootstrap.inc
# for settings that can be changed at runtime.
<IfModule mod_php5.c>
  php_flag session.auto_start               off
  php_value mbstring.http_input             pass
  php_value mbstring.http_output            pass
  php_value max_input_vars                  10007
  php_flag mbstring.encoding_translation    off
</IfModule>
```

Should be changed to 

```
# Override PHP settings that cannot be changed at runtime. See
# settings.php and backdrop_environment_initialize() in includes/bootstrap.inc
# for settings that can be changed at runtime.
<IfModule mod_php7.c>
  php_flag session.auto_start               off
  php_value mbstring.http_input             pass
  php_value mbstring.http_output            pass
  php_value max_input_vars                  10007
  php_flag mbstring.encoding_translation    off
</IfModule>
# Same settings again for PHP 5 versions.
<IfModule mod_php5.c>
  php_flag session.auto_start               off
  php_value mbstring.http_input             pass
  php_value mbstring.http_output            pass
  php_value max_input_vars                  10007
  php_flag mbstring.encoding_translation    off
</IfModule>
```

---
PR: https://github.com/backdrop/backdrop/pull/3399

### PR DESCRIPTION
**Pull request by jenlampton**
Fixes https://github.com/backdrop/backdrop-issues/issues/4752

### CODE REVIEW

**Review by klonos — APPROVED** [review]
LGTM 👍

File: .htaccess [hunk 1]
```diff
@@ -21,11 +21,17 @@ DirectoryIndex index.php index.html index.htm
...
+  php_value mbstring.http_input             pass
+  php_value mbstring.http_output            pass
+  php_flag mbstring.encoding_translation    off
+</IfModule>
+# Same settings again for PHP 5 versions.
 <IfModule mod_php5.c>
   php_flag session.auto_start               off
   php_value mbstring.http_input             pass
   php_value mbstring.http_output            pass
-  php_value max_input_vars                  10007
```
Comment (ghost) [inline .htaccess]: Why is this line removed...?

Comment (klonos) [reply .htaccess]: ...I think that this is because @jenlampton is also working on https://github.com/backdrop/backdrop-issues/issues/3824 but yeah, this should be reverted, and the same line added to the php7 chunk.

Comment (jenlampton) [reply .htaccess]: whoopsie!

**Comment by quicksketch** [comment]
Thanks! Merged into 1.x and 1.17.x.
